pkg/utils/browser.go

```diff
@@ -0,0 +1,13 @@
+package utils
```

Comment:
very very cool
did. you check linux + mac?

Reply:
yes